governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Onboard GNO to Spark Protocol - May 15, 2023
+summary: Onboard GNO (Gnosis) to Spark Protocol with included parameters. Reduce Maker GNO-A debt ceiling to zero.
+discussion_link: https://forum.makerdao.com/t/onboarding-of-gno-to-spark/20831
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-05-15T16:00:00
+end_date: 2023-05-18T16:00:00
+---
+# Poll: Onboard GNO to Spark Protocol - May 15, 2023
+
+The Arbitration Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, May 15 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding GNO (Gnosis) to Spark Protocol.
+
+### Spark Onboarding Parameters
+
+**Token:** GNO  
+**Token Address:** 0x6810e776880C02933D47DB1b9fc05908e5386b96  
+**Oracle:** Maker Medianizer via adaptor  
+**Oracle Provider:** Chronicle  
+**Borrow Enabled:** No  
+**Collateral Enabled:** Yes  
+**[Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled:** Yes  
+**Isolation Mode Debt Ceiling:** 5 million USD  
+**Loan To Value:** 20%  
```

Comment:
Does it make sense to link Aave documentation here e.g. https://docs.aave.com/risk/v/master/asset-risk/risk-parameters#loan-to-value? Same for those below